package.json

```diff
@@ -9,12 +9,7 @@
     "type": "git",
     "url": "https://github.com/thisandagain/sentiment.git"
   },
-  "keywords": [
-    "sentiment",
-    "analysis",
-    "nlp",
-    "sentiment analysis"
-  ],
+  "keywords": ["sentiment", "analysis", "nlp", "sentiment analysis"],
```

Comment:
Please don't make changes in the `package.json` that are simply aesthetic. It makes this diff hard to read / manage.